h2/src/test/org/h2/test/db/TestFunctions.java

```diff
@@ -1497,7 +1497,7 @@ private void testToCharFromDateTime() throws SQLException {
 
         assertResult("1979-11-12 08:12:34.56", stat, "SELECT X FROM T");
         assertResult("-100-01-15 14:04:02.12", stat, "SELECT X FROM U");
-        String expected = String.format("%tb", timestamp1979).toUpperCase();
+        String expected = String.format(Locale.US, "%tb", timestamp1979).toUpperCase();
```

Comment:
this is particular, is supposed to be locale-sensitive, and was fixed recently to handle different locales, so it should not be hard-cocded to one locale - perhaps you do not have a sufficiently up to date MASTER?

Reply:
It fails on current master because format() returns Nov. (with trailing period). It may be related to locale providers, modern Java versions use CLDR provider instead of own one by default.

Reply:
I'd suggest stripping the trailing period in that case